src/subscription/data/handleRequestError.js

```diff
@@ -0,0 +1,129 @@
+import { logError, logInfo } from '@edx/frontend-platform/logging';
+// import { camelCaseObject } from '../../payment/data/utils';
+// TODO: clean commented code here
+
+function handleFieldErrors(errors) {
```

Comment:
What's the difference btw this file and the original `handleRequestError.js` file? Is there a need for both?

Reply:
The difference was with the error response our new server was sending the different error key names so I have requested Shafqat to update to match the old structure. Also deleted this file and using the old one. 

![Screen Shot 2023-04-19 at 6 23 20 PM](https://user-images.githubusercontent.com/5585262/233129769-58370a0d-cee9-44bc-acc7-177e658e0355.png)

Reply:
Great, thank you 👏